media-video/guvcview/guvcview-2.2.1.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Simple Qt6 or GTK+3 interface for capturing and viewing video from v4l2 devices"
+HOMEPAGE="https://guvcview.sourceforge.net/"
+SRC_URI="https://downloads.sourceforge.net/project/${PN}/source/${PN}-src-${PV}.tar.bz2"
+S="${WORKDIR}/${PN}-src-${PV}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gsl pulseaudio qt6 +sdl sfml"
+REQUIRED_USE="
+	|| ( sdl sfml )
+"
+BDEPEND="
+	sys-devel/gettext
+	virtual/pkgconfig
+"
+RDEPEND="
+	media-libs/libpng:=
+	media-libs/libv4l
+	media-libs/portaudio
+	media-video/ffmpeg:=
+	virtual/libusb:1
+	virtual/udev
+	gsl? ( sci-libs/gsl:= )
+	pulseaudio? ( media-libs/libpulse )
+	qt6? ( dev-qt/qtbase:6[gui,widgets]	)
+	!qt6? (
+		dev-libs/glib:2
+		x11-libs/gtk+:3
+	)
+	sdl? ( media-libs/libsdl2 )
+	sfml? (
+		media-libs/libglvnd
+		media-libs/libsfml:=
+	)
+"
+# linux-headers: bug 448260
+DEPEND="
+	${RDEPEND}
+	sys-kernel/linux-headers
```

Comment:
Having both linux-headers and virtual/os-headers seems wrong? It should just be os-headers, I think.

Reply:
Could be.